cranelift-codegen/src/isa/x86/enc_tables.rs

```diff
@@ -185,9 +188,11 @@ fn expand_sdivrem(
     // Start by checking for a -1 divisor which needs to be handled specially.
     let is_m1 = pos.ins().ifcmp_imm(y, -1);
     pos.ins().brif(IntCC::Equal, is_m1, minus_one, &[]);
+    pos.ins().jump(nominal, &[]);
```

Comment:
```suggestion
    pos.ins().fallthrough(nominal, &[]);
```

Reply:
After discussing with @lars-t-hansen , we are actually going to replace all the `fallthrough` by `jump`, as the branch relaxation, which is the very last pass of the compiler is supposed to automagically do that for us.  I will revert this patch locally :/